client/verta/verta/endpoint/_kafka_settings.py

```diff
@@ -38,12 +41,13 @@ class KafkaSettings(object):
 
     """
 
-    def __init__(self, input_topic, output_topic, error_topic):
+    def __init__(self, cluster_config_id, input_topic, output_topic, error_topic):
```

Comment:
As a random side note, adding this parameter to the beginning instead of the end means that any existing code that instantiates `KafkaSettings("input", "output", "error")` will start throwing

```python
TypeError: __init__() missing 1 required positional argument: 'error_topic'
```

which looks kinda confusing ("What do you mean, my error topic is right there!"), as opposed to adding the parameter to the end which will throw a marginally more directly helpful

```python
TypeError: __init__() missing 1 required positional argument: 'cluster_config_id'
```

But I don't think anyone's using this today so it's not a worry.